packages/devextreme/js/__internal/ui/m_file_uploader.ts

```diff
@@ -861,19 +864,30 @@ class FileUploader extends Editor<FileUploaderProperties> {
     this._isCustomClickEvent = false;
   }
 
-  _attachSelectFileDialogHandler(target) {
+  _selectFileDialogKeyUpHandler(e): void {
+    if (normalizeKeyName(e) === ENTER_KEY) {
+      this._selectFileDialogClickHandler();
+    }
+  }
+
+  _attachSelectFileDialogHandlers(target) {
     if (!isDefined(target)) {
       return;
     }
-    this._detachSelectFileDialogHandler(target);
-    eventsEngine.on($(target), 'click', this._selectFileDialogHandler);
+
+    this._detachSelectFileDialogHandlers(target);
+
+    eventsEngine.on($(target), 'click', this._selectFileDialogClickHandler);
+    eventsEngine.on($(target), 'keyup', this._selectFileDialogKeyUpHandler.bind(this));
```

Comment:
[refactoring]
Can we also get rid of copypaste for target wrapping?
Ans I personally prefer arrow functions to `bind` just because it's a more modern syntax, but it's up to you

Reply:
https://github.com/DevExpress/DevExtreme/pull/28947#discussion_r1954548220